scripts/docker/requirements-20.04.txt

```diff
@@ -19,3 +19,5 @@ python3-dev=3.8.2-0ubuntu2
 python3.8-venv
 cargo
 ninja-build
+patchelf
+graphviz
```

Comment:
POSIX new line

Reply:
I'd love to build a static check for this :D